### PR TITLE
Forked orbit-web into monorepo and point to it in our app

### PR DESCRIPTION
**Pull request by robert-e-davidson3**
I also consolidated our fonts and moved the images out of the `assets/` folder in our output, to match our fonts.

### CODE REVIEW

File: packages/augur-ui/src/modules/global-chat/components/global-chat.tsx [hunk 1]
```diff
@@ -29,7 +29,7 @@ export const GlobalChat = (props: GlobalChatProps) => {
               {Close}
             </button>
           </div>
-          <iframe src='https://augurproject.github.io/orbit-web/#/channel/augur' />
+          <iframe src='./chat/index.html#/channel/augur' />
```
Comment (robert-e-davidson3) [inline packages/augur-ui/src/modules/global-chat/components/global-chat.tsx:32]: important change

File: packages/augur-ui/package.json [hunk 2]
```diff
@@ -15,8 +15,8 @@
   "scripts": {
     "clean": "rm -rf build; rm -rf node_modules",
     "nuke": "yarn clean; rm yarn.lock; rm package-lock.json",
-    "dev": "node scripts/dev.js",
-    "build": "webpack",
+    "dev": "yarn workspace orbit-web build && node scripts/dev.js",
```
Comment (robert-e-davidson3) [inline packages/augur-ui/package.json:18]: important change

File: packages/augur-ui/config/webpack.common.config.js [hunk 3]
```diff
@@ -137,7 +140,11 @@ module.exports = {
       {
         from: path.resolve(PATHS.APP, "robots.txt"),
         to: PATHS.BUILD
-      }
+      },
+      {
+        from: path.resolve(PATHS.ORBIT, "dist"),
```
Comment (robert-e-davidson3) [inline packages/augur-ui/config/webpack.common.config.js:145]: important change

**Review by ghost — COMMENTED** [review]
This pull request has been automatically deployed to [FeaturePeek](https://featurepeek.com). 🤘

 This link will always point to this PR's latest changes: https://peek.run/kq3h264

**Review by bthaile — APPROVED** [review]
Lgtm

**Pull request merged**